x-pack/plugins/maps/public/classes/fields/ems_file_field.ts

```diff
@@ -30,12 +30,6 @@ export class EMSFileField extends AbstractField implements IField {
   }
 
   async getLabel(): Promise<string> {
-    const emsFileLayer = await this._source.getEMSFileLayer();
```

Comment:
Moved login into `this._source.getEmsFieldLabel` since mocking `EMSFileSource` in ems_file_source.test was a mess with typescript and this was a code smell that we were exposing the logic of extracting the label to consumers of the source instead of encapsulating logic directly in the source.